examples/npm_lib/tests/wasm.rs

```diff
@@ -0,0 +1,27 @@
+use js_sys::Promise;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::{future_to_promise, JsFuture};
+use wasm_bindgen_test::*;
+
+use super::*;
+
+use crate::js::browser_js::run_npm_lib;
+
+async fn onclick_pjs_process_async() -> Result<wasm_bindgen::JsValue, wasm_bindgen::JsValue> {
+    run_npm_lib().await
+}
```

Comment:
What's the point of this?

Reply:
i've removed it, not necessary like you mentioned